index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <h1 id="title">This is Zakariya & Usama Tribute Page</h1>
+      <div id="img-div">
+        <img class="center" id="image" src="#" alt="Zack&Usama">
+        <div id="img-caption">Hi, we are Zakariya & Usama</div>
+        <div class="center" id="tribute-info">Hi Microverse</div>
+        <br>
+        <p id="img-caption">Usama The Lion & Zakariya The Programmer</p>
```

Comment:
- [ ] Required: Please use the `id="img-caption"`  only for one element. According to **[CSS Best Practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)** each id value must be used only once within a document. If more than one element has been assigned the same `ID`, queries that use that `ID` will only select the first matched element in the `DOM`. This behavior should not be relied on, however; a document with more than one element using the same ID is **invalid**.